### PR TITLE
Review

### PR DESCRIPTION
**Pull request by choongzhanhong**
Add review capabilities.

### CODE REVIEW

File: src/main/java/Review.java [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    /**
+     * Sets the star review from 0 to 5 (Inclusive).
+     * Feedback to user (No change) if stars are not valid.
+     * @param stars star rating of 0 to 5 inclusive.
+     */
+    public void setReviewStars(int stars) {
+        // Bad path
+        if (stars > 5 || stars < 0) {
+            System.out.println("Hey! Reviews only from 0 to 5!");
+            // Throw MovieMate exception?
```
Comment (chao2048) [inline src/main/java/Review.java:61]: I agree with the comment, an exception should be thrown here

File: src/main/java/Review.java [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+                if (stars > 5 || stars < 0) {
+                    System.out.println("Hey! Reviews only from 0 to 5!");
+                    // Throw MovieMate exception?
+                    continue;
+                }
+                this.reviewStars = stars;
+                return;
+            } else {
+                System.out.println("Please input a number [0 to 5]!");
+            }
```
Comment (chao2048) [inline src/main/java/Review.java:83]: Continue with the previous part, if exception is used then perhaps the nesting will be solved?

**Review by chao2048 — COMMENTED** [review]
Overall LGTM, thanks for the effort and the nice OOP practice!

File: src/main/java/Review.java (hunk 1 above)
Comment (choongzhanhong) [reply src/main/java/Review.java:61]: Actually, now that I think about it, I think it's fine to not throw an exception as the issue can be handled here.

**Pull request merged**